python/ray/util/state/common.py

```diff
@@ -89,8 +89,14 @@ class Humanify:
     """A class containing default methods to
     convert units into a human readable string."""
 
-    def timestamp(x: float):
-        """Converts miliseconds to a datetime object."""
+    def timestamp(x):
+        """
+        Converts milliseconds to a datetime object.
+        If the input is -1, return an empty string.
+        If the input is not an int or float, raise a ValueError.
+        """
+        if x == -1:
+            return ""
```

Comment:
I think we should do it on the caller side as the caller knows what special value can be so that this method doesn't need to handle -1, None or some other special values

Reply:
fixed